common/religion/religions/light_group.txt

```diff
@@ -24,6 +24,20 @@
 	doctrine = doctrine_deviancy_crime
 	doctrine = doctrine_witchcraft_crime
 
+	#Magic
+	doctrine = doctrine_light_magic_approved
+	doctrine = doctrine_shadow_magic_crime
+	doctrine = doctrine_disorder_magic_crime
+	doctrine = doctrine_order_magic_shunned
```

Comment:
Order should not be shunned but ignored / accepted instead. Human were taught Order magic to won the trolls war. Even Fundamentalists such as Scarlets were using lots of mages:
https://wow.gamepedia.com/Arcanist_Doan

Reply:
Yeah, tbh I'm not sure myself

> Magi have been considered heretics for centuries by the Church of the Light who has never been fond of their methods, but let them continue to exist because they don't practice dark magic and are somewhere necessary.

https://wow.gamepedia.com/Heretic

> It was too much. "We've been called heretics for centuries. The church has never been fond of our methods. Such sentiments notwithstanding, we are still here."
> She nodded. "Because we avoid dark magic, which leads to corruption and catastrophe."
> "Because we are necessary!"

https://wow.gamepedia.com/Road_to_Damnation

Reply:
Ok, I found another reference for it.
> Some Knights of the Silver Hand believed that after death, a mage's soul would be damned.

https://wow.gamepedia.com/Light

So yeah I think you are right. Mage was shunned for at least a while.